src/legacy/containers/Root.tsx

```diff
@@ -22,7 +22,7 @@ const Root = (): JSX.Element => {
    * Remove this CONST when new designs will be applied.
    * For now you can switch between old and new application by changing SHOW_ONBOARDING
    */
-  const SHOW_ONBOARDING = false
+  const SHOW_ONBOARDING = true
```

Comment:
Please set it to false, as other developers work on other parts and will spend some time to find this constant to set it to false.